check/shellcheck

```diff
@@ -30,11 +30,11 @@ for arg in "$@"; do
 done
 
 # Find all shell scripts in this repository.
-IFS=$'\n' read -r -d '' -a our_shell_scripts < \
-    <(git ls-files -z -- \
+IFS=$'\n' read -r -d '' -a our_shell_scripts <<< \
+    "$(git ls-files -z -- \
```

Comment:
Nit - bash manual suggest `<<<word` syntax for the HERE strings
```suggestion
IFS=$'\n' read -r -d '' -a our_shell_scripts <<<"$(
    git ls-files -z -- \
```

Also, can you please remove unnecessary backslash after pipe on line 36?